code/math/vecmat.cpp

```diff
@@ -1397,7 +1397,7 @@ void compute_point_on_plane(vec3d *q, const plane *planep, const vec3d *p)
 
 	vm_vec_scale_add(q, p, &normal, -k);
 
-	tv = planep->A * q->xyz.x + planep->B * q->xyz.y + planep->C * q->xyz.z + planep->D;
+	// tv = planep->A * q->xyz.x + planep->B * q->xyz.y + planep->C * q->xyz.z + planep->D;
```

Comment:
Refer https://github.com/scp-fs2open/fs2open.github.com/issues/229. The changes to `compute_point_on_plane()` either need (a) an accompanying documentation change, or (b) review of the refactor.

I'm rather against just commenting out the offending line, as there could be an implementation oversight here.